df.world-data.xml

```diff
@@ -703,18 +703,20 @@
         <int32_t name="year_tick"/>
 
         -- v0.40.14:
-        <stl-vector name='unk_c0' type-name='int16_t' comment="6 bytes allocated seen. Cannot be pointer"/>
-        <stl-vector name='unk_d0' type-name='int16_t' comment="6 bytes allocated seen. Cannot be pointer"/>
-        <stl-vector name='unk_e0' type-name='int16_t' comment="6 bytes allocated seen. Cannot be pointer. Seen on the same element, so c0, d0, and e0 may be connected"/>
-        <stl-vector name='unk_f0' type-name='int32_t' comment="4 bytes allocated seen. Cannot be pointer"/>
-        <stl-vector name='unk_100' type-name='int32_t' comment="4 bytes allocated seen. Cannot be pointer"/>
-        <stl-vector name='unk_110' type-name='int32_t' comment="4 bytes allocated seen. Cannot be pointer. Seen on the same element, so f0, 100, and 110 may be connected"/>
+        <compound name='picked_growths' comment="also includes 'automatically picked' i.e. fallen fruit that becomes item_spatter">
```

Comment:
Might be good to move the version into a `since` attribute:
```suggestion
        <compound name='picked_growths' since='v0.40.14' comment="also includes 'automatically picked' i.e. fallen fruit that becomes item_spatter">
```